packages/node/migrations.json

```diff
@@ -45,7 +45,7 @@
     "update-16-3-1-update-executor": {
       "cli": "nx",
       "version": "16.3.1-beta.0",
-      "description": "Replace @nx/node:webpack with @nx/node:webpack for all project targets",
+      "description": "Replace @nrwl/node:webpack with @nx/node:webpack for all project targets",
```

Comment:
Thank you for fixing this. However, this is the wrong message as well.

Both `@nrwl/node:webpack` and `@nx/node:webpack` are replaced with `@nx/webpack:webpack`.

Could you update it please?

Reply:
@FrozenPandaz updated